source/packages/ui-library/src/components.d.ts

```diff
@@ -3325,15 +4082,15 @@ declare namespace LocalJSX {
         /**
           * Set the options to be shown in the dropdown
          */
-        "items"?: SixMenuItemData[] | null;
+        "items"?: SixMenuItemData1[] | null;
```

Comment:
Is there a particular reason why you renamed this to SixMenuItemData1 or is this a typo?

Reply:
This file is generated by the stencil compiler. Seems that stencil 3 does not output the exact same code as stencil 2. The semantics should be the same though.
In this particular case, stencil aliases `SixMenuItemData` with `import { SixMenuItemData as SixMenuItemData1...`.

Reply:
Ok in that case if it's just an alias we should be good.